.github/workflows/tests.yml

```diff
@@ -7,7 +7,7 @@ jobs:
   validate:
     strategy:
       matrix:
-        ruby-version: ["2.6", "2.7", "3.0", "3.1", "3.2"]
+        ruby-version: ["2.7", "3.0", "3.1", "3.2"]
```

Comment:
2.7 is also out of live right?

Reply:
Yeah but this should still work with 2.7